db.js

```diff
@@ -538,52 +551,78 @@ export class ActivityPubDB extends EventTarget {
   }
 }
 
-function upgrade (db) {
-  const actors = db.createObjectStore(ACTORS_STORE, {
-    keyPath: 'id',
-    autoIncrement: false
-  })
-
-  actors.createIndex(CREATED_FIELD, CREATED_FIELD)
-  actors.createIndex(UPDATED_FIELD, UPDATED_FIELD)
-  actors.createIndex(URL_FIELD, URL_FIELD)
-
-  db.createObjectStore(FOLLOWED_ACTORS_STORE, {
-    keyPath: 'url'
-  })
-
-  const notes = db.createObjectStore(NOTES_STORE, {
-    keyPath: 'id',
-    autoIncrement: false
-  })
-  notes.createIndex(ATTRIBUTED_TO_FIELD, ATTRIBUTED_TO_FIELD, { unique: false })
-  notes.createIndex(IN_REPLY_TO_FIELD, IN_REPLY_TO_FIELD, { unique: false })
-  notes.createIndex(PUBLISHED_FIELD, PUBLISHED_FIELD, { unique: false })
-  addRegularIndex(notes, TO_FIELD)
-  addRegularIndex(notes, URL_FIELD)
-  addRegularIndex(notes, TAG_NAMES_FIELD, { multiEntry: true })
-  addSortedIndex(notes, IN_REPLY_TO_FIELD)
-  addSortedIndex(notes, ATTRIBUTED_TO_FIELD)
-  addSortedIndex(notes, CONVERSATION_FIELD)
-  addSortedIndex(notes, TO_FIELD)
-
-  const activities = db.createObjectStore(ACTIVITIES_STORE, {
-    keyPath: 'id',
-    autoIncrement: false
-  })
-  activities.createIndex(ACTOR_FIELD, ACTOR_FIELD)
-  addSortedIndex(activities, ACTOR_FIELD)
-  addSortedIndex(activities, TO_FIELD)
-  addRegularIndex(activities, PUBLISHED_FIELD)
+async function migrateNotes (db) {
```

Comment:
Pass the transaction from the `update` function in here and use it instead of `db.transaction`